CONTRIBUTING.md

```diff
@@ -83,13 +101,50 @@ npm run dev:pristine
 npm run dev:format
 ```
 
+## Coding conventions
+
+1. We recommend small, single-purpose components. We recommend breaking up large components into smaller, more cohesive components. Smaller components are easier to test because you can isolate functionality to ensure that components are immune to changes in other components. In general, larger components lead to larger tests that are more difficult to write and more difficult to understand.
+1. Use the `Sky` prefix when naming all classes, directives, services, components, etc. The prefix indicates to other contributors that items are owned by SKY UX and not a third-party library. It also prevents potential class-name clashes with other libraries. Keep in mind that while we generally use the uppercase `Sky` prefix, we also use the `sky-` prefix in some cases, such as selector properties in components.
+
+## Test harnesses
+
+When you create components, include a test harness so that consumers can use them in unit tests to query against the components. These harnesses should be placed in the library's `testing` folder and should be exported via the `testing/public-api.ts` file. These harnesses should extend the `SkyComponentHarness` class which extends the Angular CDK `ComponentHarness` class.
+
+The test harness should give consumers the ability to query the harness for information about the component which may rely on the component's internal structure to determine. This allows consumers to avoid writing brittle tests that may break due to internal changes within a major version.
+
+All items in the `testing` folder should be fully unit tested with same standards which apply to the main component library.
+
+## Unit tests
+
```

Comment:
In the docs site (https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#unit-tests), we have a lead-in paragraph with a recommendation to reach out to the team for a soft review and with a link to Angular testing guidelines. Should we include that paragraph here?

Reply:
That was a miss. I have added that paragraph back.